p/php-src/php-src_ubi_9.3.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package          : php-src
+# Version          : php-8.4.0alpha1
+# Source repo      : https://github.com/php/php-src
+# Tested on        : UBI:9.3
+# Language         : PHP
+# Travis-Check     : True
+# Script License   : PHP License
+# Maintainer       : Vinod K <Vinod.K1@ibm.com>
+#
+# Disclaimer       : This script has been tested in root mode on given
+# ==========         platform using the mentioned version of the package.
+#                    It may not work as expected with newer versions of the
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=php-src
+PACKAGE_VERSION=${1:-php-8.4.0alpha1}
+PACKAGE_URL=https://github.com/php/php-src
+
+OS_NAME=$(grep ^PRETTY_NAME /etc/os-release | cut -d= -f2)
+
+yum install -y git wget expect
+dnf config-manager --add-repo https://mirror.stream.centos.org/9-stream/AppStream/ppc64le/os/
+dnf config-manager --add-repo https://mirror.stream.centos.org/9-stream/BaseOS/ppc64le/os/
+dnf config-manager --add-repo https://mirror.stream.centos.org/9-stream/CRB/ppc64le/os/
+dnf install --nodocs -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+wget http://mirror.centos.org/centos/RPM-GPG-KEY-CentOS-Official
+mv RPM-GPG-KEY-CentOS-Official /etc/pki/rpm-gpg/.
+rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official
+
+dnf install -y re2c bison autoconf make libtool ccache libxml2-devel sqlite-devel glibc-gconv-extra pcre expect
+
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+./buildconf --force
```

Comment:
is this build kind of command.. if yes lets track exit code as 1 for failure in both buildconf and configure step

Reply:
@nikhil-kalbande  updated the script as suggested